memgpt/cli/cli.py

```diff
@@ -516,7 +518,15 @@ def run(
 
     # determine agent to use, if not provided
     if not yes and not agent:
-        agents = ms.list_agents(user_id=user.id)
+        try:
+            agents = ms.list_agents(user_id=user.id)
+        except sqlalchemy.exc.OperationalError as e:
```

Comment:
@sarahwooders FYI not sure if this is (1) the right place to put this (2) if we need to add more checks instead of just the `if "no such column" in str(e)`, but it's an example and it worked locally (so maybe just merge?)